README.md

```diff
@@ -1,2 +1,307 @@
+
+Medication classes:
+
+|Medication Class|Code|
+|-|-|
+
+Medications:
+
+|Medication|Code|Alternative|
+|-|-|-|
+
+### /users/$userId$/medicationRequests/$medicationRequestId$
+
+Based on [FHIR MedicationRequest](https://hl7.org/fhir/R4B/medicationrequest.html), the following properties may be used, while additional properties are ignored by the Engage-HF system.
+
+|Property|Type|Values|Comments|
+|-|-|-|-|
+|id|string|-|[Resource](https://hl7.org/fhir/R4B/resource.html): Logical id of this artifact|
+|medication|Reference(Medication) or CodeableConcept|-|CodeableConcept containing one of the codes from /medications/$medicationId$|
+|dosageInstruction|Dosage|-|-|
+|patient|string|e.g. "allergy", "intolerance"|`userId` as used in /users/$userId$ and related collections.|
+
+The `dosageInstruction` property may contain values with the following properties:
+
+|Property|Type|Values|Comments|
+|-|-|-|-|
+|id|string|-|[Resource](https://hl7.org/fhir/R4B/resource.html): Logical id of this artifact|
+|text|optional string|-|Free text dosage instructions|
+|additionalInstruction|list of CodableConcept|information like "with meals", "may cause drowsiness", etc|Supplemental instruction or warnings to the patient|
+|patientInstruction|optional string|-|Patient or consumer oriented instructions|
+|timing|optional [Timing](https://hl7.org/fhir/R4B/datatypes.html#timing)|-|When medication should be administered|
+|doseAndRate|list of Element|-|Amount of medication administered|
+|doseAndRate>type|optional [DoseRateType](https://hl7.org/fhir/R4B/codesystem-dose-rate-type.html)|e.g. "calculated", "ordered"||
+|doseAndRate>dose|optional [SimpleQuantity](https://hl7.org/fhir/R4B/datatypes.html#SimpleQuantity)|-|TBD: We should either use "pills" or some weight unit here.|
+|maxDosePerPeriod|optional [Ratio](https://hl7.org/fhir/R4B/datatypes.html#ratio)|-|Upper limit on medication per unit of time|
+|maxDosePerAdministration|optional [SimpleQuantity](https://hl7.org/fhir/R4B/datatypes.html#SimpleQuantity)|-|Upper limit on medication per administration|
+|maxDosePerLifetime|optional [SimpleQuantity](https://hl7.org/fhir/R4B/datatypes.html#SimpleQuantity)|-|Upper limit on medication per unit of time|
+
+There may be up to three intakes per day (morning, mid-day and evening) with either 0.5, 1 or 2 pills. The dosages should always be grouped by medication, i.e. there should not be multiple medication elements concerning the same medication but different dosage instructions. Instead, one medication element shall be used for that medication with multiple dosage instructions.
+
+### /users/$userId$/observations/$observationId$
```

Comment:
I have discussed a few things with @nriedman as well and we would align that it makes sense to store weight scale measurements under `/users/$userId$/bodyWeight/$observationId$` and blood pressure `/users/$userId$/bloodPressure/$observationId$` to make it easier to query and also save the observations. All the other structures should stay the same.

Reply:
okay, could we use `/users/$userId$/observations/bodyWeight/$observationId$` and `/users/$userId$/observations/bloodPressure/$observationId$` instead?

Reply:
✅